xtask/src/swift.rs

```diff
@@ -0,0 +1,85 @@
+use std::fs;
+
+use clap::{Args, Subcommand};
+use xshell::{cmd, pushd};
+
+use crate::{workspace, Result};
+
+#[derive(Args)]
+pub struct SwiftArgs {
+    #[clap(subcommand)]
+    cmd: Option<SwiftCommand>,
+}
+
+#[derive(Subcommand)]
+enum SwiftCommand {
+    /// Build as a static lib.
+    BuildLibrary,
+    /// Build as an XCFramework.
+    BuildFramework,
+}
+
+impl SwiftArgs {
+    pub fn run(self) -> Result<()> {
+        let _p = pushd(&workspace::root_path()?)?;
+
+        match self.cmd {
+            Some(cmd) => match cmd {
+                SwiftCommand::BuildLibrary => build_library(),
+                SwiftCommand::BuildFramework => build_xcframework(),
+            },
+            None => build_library(),
```

Comment:
The common patter in the CLI is that if no subcommands are given, all commands are run. Why are opting for only one. Either give some docs about this in the CLI or switch this to running both, please:
```suggestion
            None => {
                build_library()?;
                build_xcframework()
            }
```